### ISSUE TITLE
cannot debug nodejs16 (node 16) lambda in local docker

### ISSUE THREAD

**Issue by justinmk3** [aws/aws-sam-cli#3916]
### Description:

Looks like `index.js` is missing in the nodejs Lambda image, and that is the expected entrypoint for local Docker-based Lambda debugging: https://github.com/aws/aws-sam-cli/blob/d7e0c438a2853cc1618f82bfe2828acdc97df7a0/samcli/local/docker/lambda_debug_settings.py#L137

```
docker run -it --entrypoint=bash public.ecr.aws/lambda/nodejs:16`
bash-4.2# cat /var/runtime/index.js
cat: /var/runtime/index.js: No such file or directory
```

Related:

- https://github.com/aws/aws-lambda-base-images/issues/14#issuecomment-1120864028
- https://github.com/aws/aws-sam-cli/issues/2573
- https://github.com/aws/aws-sam-cli/pull/3913
- #3886

### Steps to reproduce:

```
sam build
sam local invoke --debug HelloWorldFunction --template template.yaml -d 5861
```

Try to attach a nodejs debugger to port 5861.

### Observed result:

debugger fails to attach:

```
Error: Cannot find module '/var/runtime/index.js'
    at Function._resolveFilename (node:internal/modules/cjs/loader:933:15)
    at Function._load (node:internal/modules/cjs/loader:778:27)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:77:12)
    at node:internal/main/run_main_module:17:47 {
  code: 'MODULE_NOT_FOUND',
  requireStack: []
}
27 May 2022 16:59:01,657 [ERROR] (rapid) Init failed error=Runtime exited with error: exit status 1 InvokeID=
```

### Expected result:

can attach nodejs debugger.

### Additional environment details (Ex: Windows, Mac, Amazon Linux etc)

1. OS: macos
3. `sam --version`: 1.50
4. AWS region: n/a

**Comment by justinmk3** [comment]
as a workaround (or maybe it's the proper fix?) here is a PR for sam cli (as opposed to changing the Lambda image): https://github.com/aws/aws-sam-cli/pull/3913

**Comment by mingkun2020** [comment]
Thanks @justinmk3 for submitting the issue, does the above fix working for this issue?

**Comment by mndeveci** [comment]
This is been released with [SAM CLI v1.51.0](https://github.com/aws/aws-sam-cli/releases/tag/v1.51.0)
